hpedockerplugin/hpe_storage_api.py

```diff
@@ -332,9 +447,9 @@ def volumedriver_create(self, name, opts=None):
 
         # Verify valid Opts arguments.
         valid_volume_create_opts = ['mount-volume', 'compression',
-                                    'size', 'provisioning', 'flash-cache']
-
-        valid_compression_opts = ['true', 'false']
+                                    'size', 'provisioning', 'flash-cache',
+                                    'cloneOf', 'snapshotOf', 'expirationHours',
+                                    'retentionHours']
 

```

Comment:
- We need to validate if the -o cloneOf=<clonename> -o snapshotOf=<snapshot_name> is not present as part of same $ docker volume create option, they should be used mutually exclusive